### ISSUE TITLE
no debug logs

### ISSUE THREAD

**Issue by vanvank** [bfren/docker-ftps#143]
when I set BF_FTPS_VSFTPD_ENABLE_DEBUG_LOG=1， 
nothing happens when I monitoring logs by :  docker logs -f  <ftps_container_name>

**Comment by vanvank** [comment]
in the container,the /var/log/vsftpd.log has no content

**Comment by bfren** [comment]
Yes the feature is not working currently, I've not been able to work out why but I'm investigating!

**Comment by bfren** [comment]
OK so I think I've cracked it - got it working now locally, going to test on my live server, but by all means give it a try - pull the `:dev` tag.
